components/experimental/src/dimension/provider/currency_patterns.rs

```diff
@@ -52,3 +53,17 @@ pub enum PatternCount {
     /// UnitPattern `other`.
     Other = 5,
 }
+
+impl From<PluralCategory> for PatternCount {
```

Comment:
why do I keep seeing conversions between `Count` enums? You're adding two (more?) here, is that necessary?

Reply:
Yes, one from `PatternCount` and the other for `Count` for currency, they are different counts with different fields.

This something we need to generalize across ICU4X soon

Reply:
This one seems to be identical, there are no extra fields

Reply:
https://github.com/unicode-org/icu4x/issues/5369

Reply:
https://github.com/unicode-org/icu4x/pull/5373

Reply:
done